docs/src/pages/guides/examples.md

```diff
@@ -214,3 +214,56 @@ probes:
 ```
 
 Using the above configuration, Monika will perform a login request in the first request, then use the returned token in the Authorization header of the second request.
+
+### Pass Response Data to Request Body
+
+Continuing with the examples from wwwh.reqres.in above, say we would like to use the previous GET request to perform a POST /login. If the initial data from reqres returned is something like below:
+
+```yaml
+{
+  'data':
+    {
+      'id': 1,
+      'email': 'george.bluth@reqres.in',
+      'first_name': 'George',
+      'last_name': 'Bluth',
+      'avatar': 'https://reqres.in/img/faces/1-image.jpg',
+    },
+  ....,
+}
+```
+
+Then your chain request to get the user body.email would look something like:
+
+```yaml
+probes:
+  - id: probe-01
+    name: 'body from response'
+    interval: 10
+
+    requests:
+      - url: https://reqres.in/api/users/1
+        method: GET
+        timeout: 5000
+        saveBody: false
+        headers:
+          Content-Type: application/json; charset=utf-8
+
+      - url: https://reqres.in/api/login
+        method: POST
+        timeout: 1000
+        headers:
+          Content-Type: application/json; charset=utf-8
+        body:
+          email: '{{ responses.[0].body.data.email }}'
+          password: password
+
+        alerts:
+          - query: response.status != 200
+            message: Http Response status code is not 200!
+notifications:
+  - id: unique-id-desktop
+    type: desktop
+```
+
+Note: Please do not forget the tick marks before and after the opening and closing double braces to explicitly indicate a string value. YAML parsers will generate warnings without it.
```

Comment:
i think it's better to call it `single quote` instead of tick mark

Reply:
fixed!